common/src/main/java/com/microsoft/hydralab/performance/parsers/IOSEnergyGaugeResultParser.java

```diff
@@ -82,4 +84,67 @@ public PerformanceTestResult parse(PerformanceTestResult performanceTestResult)
         }
         return performanceTestResult;
     }
+
+    private IOSEnergyGaugeInfo getAverageIOSEnergy(List<PerformanceInspectionResult> inspectionResults) {
+        if (inspectionResults == null || inspectionResults.size() == 0) {
+            return null;
+        }
+
+        IOSEnergyGaugeInfo sumEnergyInfo = new IOSEnergyGaugeInfo();
```

Comment:
It seems that this variable holds the average and not the sum. How about `averageEnergyInfo`

Reply:
Nice catch. I've revised